Objective-C/CBLIndexable.h

```diff
@@ -28,21 +28,53 @@ NS_ASSUME_NONNULL_BEGIN
 /** The Indexable interface defines a set of functions for managing the query indexes. */
 @protocol CBLIndexable <NSObject>
 
-/** Return all index names, or nil if an error occurred. */
+/**
+ Get the names of all indexes of the collection.
+ 
+ @param error On return, the error if any.
+ @return Names of all indexes of the collection or nil if no indexes exist.
```

Comment:
If no indexes exists, I think it will return an empty array (please double check), only when there is an error, it will return nil.